### ISSUE TITLE
fresh install

### ISSUE THREAD

**Issue by KraftAffix** [LemmusLemmus/GlosSI-Integration-Playnite#25]
![image](https://github.com/LemmusLemmus/GlosSI-Integration-Playnite/assets/69615794/06b1b282-469a-43e3-bc33-3c1913df355e)

**Comment by LemmusLemmus** [comment]
Does the folder `%appdata%\GlosSI` (i.e. `C:\Users\<your-username>\AppData\Roaming\GlosSI`) exist? And does it contain a `Targets` folder?

**Comment by KraftAffix** [comment]
> Does the folder `%appdata%\GlosSI` (i.e. `C:\Users\<your-username>\AppData\Roaming\GlosSI`) exist? And does it contain a `Targets` folder?

theres no folder named GlosSI in my `%appdata%\GlosSI`

**Comment by LemmusLemmus** [comment]
Does `%appdata%\GlosSI` not exist? Try launching `GlosSIConfig.exe` once (no need to do anything there, you can simply close it afterwards) and the folders should be created. Does this solve the problem?

**Comment by KraftAffix** [comment]
> Does `%appdata%\GlosSI` not exist? Try launching `GlosSIConfig.exe` once (no need to do anything there, you can simply close it afterwards) and the folders should be created. Does this solve the problem?

Where is this located? I can only find `config.json` and `DefaultTarget.json` in the extensions data folder
(`C:\Users\<username>\AppData\Roaming\Playnite\ExtensionsData\6b0297da-75e5-4330-bb2d-b64bff22c315`)

**Comment by LemmusLemmus** [comment]
It should be located where you installed/downloaded [GlosSI](https://glossi.flatspot.pictures/).

**Comment by Vylyne** [comment]
I just wanted to put out there my experience as I had the same issue come up. Where the %AppData%\GlosSi Folder was not created and hence no Targets folder in it and was getting this error.

I Installed both Playnite and GlosSi through Chocolatey

When Installed this way GlosSi did not get shortcuts anywhere so I had to go digging. At the time of writing this post Chocolatey installs GlosSi to %ProgramData%\chocolatey\lib\glossi\tools and puts shims in %ProgramData%\chocolatey\bin, and I needed to manually add the GlosSiTarget.exe to Steam as an Non-Steam Game/App and run the GLosSiConfig.exe once to get the %AppData%\GlosSi folder created.

None of this is really an issue or even something that this integration should really need to address. Though maybe we could add the Run GlosSiConfig.exe and make sure %AppData%\GlosSi and %AppData%\GlosSi\Targets exist on a windows machine.

TLDR;
Running the **%ProgramData%\chocolatey\bin\GlosSiConfig.exe** created the **%AppData%\GlosSi** and resolved the error for me.

Installed Playnite and GlosSi through Chocolatey.
GlosSi shims are located at **%ProgramData%\chocolatey\bin** ([more info on shims](https://docs.chocolatey.org/en-us/features/shim))
GlosSi actual binaries are located at %ProgramData%\chocolatey\lib\glossi\tools (shims should be all that's necessary)

**Comment by LemmusLemmus** [comment]
If the `%appdata%\GlosSI` folder is missing, the extension should probably be able to simply create the folder and everything should hopefully work. To make the startup process less painful, I should also fix #20. This should be easy enough.

More problematic is the fact that the extension does not know where GlosSI is located. Currently the extension just checks, IIRC, the directories `Program Files\GlosSI` and `Program Files (x86)\GlosSI`. I should definitely improve this.

Thanks for providing useful info about GlosSI with Chocolatey! I'll be sure to update the extension to automatically find that path as well.

**Comment by LemmusLemmus** [comment]
@LucasBrunton BTW, is the Chocolatey version of GlosSI not outdated, or has it been updated?

**Comment by Vylyne** [comment]
You know, I didn't even think about the version of GlosSi in chocolatey. That's one of the things I hate about chocolatey everything is always out of date... Just checked looks like it's currently at version 0.0.8.1. So guess I'll be switching.
